src/utils/media-utils.js

```diff
@@ -438,7 +438,7 @@ export function spawnMediaAround(el, media, snapCount, mirrorOrientation = false
   return { entity, orientation };
 }
 
-const hubsSceneRegex = /https?:\/\/(hubs.local(:\d+)?|(smoke-)?hubs.mozilla.com)\/scenes\/(\w+)\/?\S*/;
+const hubsSceneRegex = /https?:\/\/((dev.reticulum.io)|(hubs.local(:\d+)?|(smoke-)?hubs.mozilla.com))\/scenes\/(\w+)\/?\S*/;
```

Comment:
i don't think we want to commit this change -- this would mean links to dev scenes would be recognized as scene links in production. if we want to get this working in dev it should be conditional at some point i think